tests/TestCase/Command/I18nExtractCommandTest.php

```diff
@@ -273,7 +273,7 @@ public function testExtractPlugin(): void
      */
     public function testExtractVendorPrefixedPlugin(): void
     {
-        $this->loadPlugins(['Company/TestPluginThree']);
+        $this->setApplicationPlugins(['Company/TestPluginThree']);
```

Comment:
Would we expect application devs and plugin maintainers to switch to the new method? These kinds of changes make upgrading harder as there isn't a simple way to use the new method in the old release and have a smooth upgrade. Could `loadPlugins()` modify or replace ` $this->appPluginsToLoad`?

Reply:
> Could loadPlugins() modify or replace  $this->appPluginsToLoad?

Hmm.. I'll look into it.